src/frontends/pytorch/src/op/reshape.cpp

```diff
@@ -13,25 +13,8 @@ namespace pytorch {
 namespace op {
 
 OutputVector translate_reshape(NodeContext& context) {
```

Comment:
Please add comment here that view has the same semantics. Also would be very nice to have schema in comments for both reshape and view

Reply:
Done